src/inner_connection.rs

```diff
@@ -14,7 +14,6 @@ use crate::error::{error_from_handle, error_from_sqlite_code, Error};
 use crate::raw_statement::RawStatement;
 use crate::statement::Statement;
 use crate::unlock_notify;
-use crate::version::version_number;
```

Comment:
Due to conditional compilation, `version_number` is not referenced when both `bundled` and `loadable_extension`, so its use here triggers a lint. It has been moved into the blocks where it is actually used.